File: app/src/main/java/com/akameko/testforlifehackstudio/repository/CompanyDetailItem.java
package com.akameko.testforlifehackstudio.repository;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class CompanyDetailItem {
    @SerializedName("id")
    @Expose
    private Integer id;
    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("img")
    @Expose
    private String img;
    @SerializedName("description")
    @Expose
    private String description;
    @SerializedName("lat")
    @Expose
    private String lat;
    @SerializedName("lon")
    @Expose
    private String lon;
    @SerializedName("www")
    @Expose
    private String www;
    @SerializedName("phone")
    @Expose
    private String phone;

    public CompanyDetailItem(Integer id, String name, String img, String description, String lat, String lon, String www, String phone) {
        this.id = id;
        this.name = name;
        this.img = img;
        this.description = description;
        this.lat = lat;
        this.lon = lon;
        this.www = www;
        this.phone = phone;
    }

    public Integer getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getImg() {
        return img;
    }

    public String getDescription() {
        return description;
    }

    public String getLat() {
        return lat;
    }

    public String getLon() {
        return lon;
    }

    public String getWww() {
        return www;
    }

    public String getPhone() {
        return phone;
    }
}

File: app/src/main/java/com/akameko/testforlifehackstudio/repository/CompanyDetailItem.kt
package com.akameko.testforlifehackstudio.repository

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class CompanyDetailItem(@field:Expose @field:SerializedName("id") val id: Int, @field:Expose @field:SerializedName("name") val name: String, @field:Expose @field:SerializedName("img") val img: String, @field:Expose @field:SerializedName("description") val description: String, @field:Expose @field:SerializedName("lat") val lat: String, @field:Expose @field:SerializedName("lon") val lon: String, @field:Expose @field:SerializedName("www") val www: String, @field:Expose @field:SerializedName("phone") val phone: String) {

    override fun toString(): String {
        return "CompanyDetailItem{" +
                "id=" + id +
                ", name='" + name + '\'' +
                ", img='" + img + '\'' +
                ", description='" + description + '\'' +
                ", lat='" + lat + '\'' +
                ", lon='" + lon + '\'' +
                ", www='" + www + '\'' +
                ", phone='" + phone + '\'' +
                '}'
    }

}